health/api/src/main/java/com/cerner/beadledom/health/api/DependenciesResource.java

```diff
@@ -45,12 +44,12 @@
       @io.swagger.annotations.ApiResponse(code = 200, message = "healthy",
           response = HealthDependencyDto.class)})
   @GET
-  @Produces(MediaType.APPLICATION_JSON)
+  @Produces("application/json; qs=0.9")
```

Comment:
Any thoughts on a good way to create a single source of truth of the 2 supported media types for health check endpoints?

The only idea occurring to me is to define them as static fields in a private class in this same package.

Reply:
That's all I could also come up with. Created MediaType class, 98f5dc0f